### PR TITLE
Add xgboost gallery demo

### PR DESCRIPTION
**Pull request by sauyon**
Has a few minor copyediting fixes that should probably be ported to the quickstart as well.

### CODE REVIEW

**Comment by sauyon** [comment]
Updated for 1.0 runner API. Should work with bentoml/BentoML#2459 and bentoml/BentoML#2461 merged.

File: xgboost/README.md [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+your local docker environment to build a new docker image containing the Bento:
+
+```bash
+bentoml containerize agaricus:latest
+```
+
+To test out the newly created docker image:
+
+```bash
+docker run agaricus:<bento tag output> -p 5000:5000
```
Comment (aarnphm) [inline xgboost/README.md:175]:
```suggestion
docker run agaricus:<bento tag output> -p 3000:3000
```

File: xgboost/README.md [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+```bash
+bentoml models list agaricus
+```
+
+Verify that the model can be loaded as runner from Python shell:
+
+```python
+import bentoml
+
+runner = bentoml.xgboost.load_runner("agaricus:latest")
```
Comment (parano) [inline xgboost/README.md:32]: we will need `to_runner` and `init_runner` now, could we make this part consistent with? https://github.com/bentoml/gallery/tree/runner-1.0/quickstart

Comment (parano) [inline xgboost/README.md:32]: we will need `to_runner` and `init_runner` now, could we make this part consistent with? https://github.com/bentoml/gallery/tree/runner-1.0/quickstart

**Pull request merged**